src/index.js

```diff
@@ -1,28 +1,26 @@
+// eslint-disable-next-line import/no-cycle
+import formTask from './newTasks.js';
+// eslint-disable-next-line import/no-cycle
+import renderTasks from './render.js';
```

Comment:
- [ ] You have disabled the linter check on your code. Please enable all linter checks on your code to ensure that your code is consistent and clean.

Reply:
Thank you Julie for your response all the linter is working as you can see, all the project its working 100%.